backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/serialization/KonanIr.proto

```diff
@@ -386,6 +597,6 @@ message IrStatement {
         IrBlockBody block_body = 4;
         IrBranch branch = 5;
         IrCatch catch = 6;
+        IrSyntheticBody synthetic_body = 7;
```

Comment:
What's the motivation behind including `IrBlockBody`, `IrBranch`, `IrCatch` and `IrSyntheticBody` to `IrStatement`? I suppose this should be eventually reworked, before releasing this format.

Reply:
Okay.